olwebsite-app/src/pages/LoginView.jsx

```diff
@@ -1,9 +1,92 @@
-import React from "react";
+import React, { useState } from "react";
+import { Redirect, withRouter } from "react-router-dom";
+
+import { 
+  validateEmail, 
+  validatePassword
+} from '../utils/authentication-utils';
+import { SUCCESS } from '../constants/authentication-constants';
 
 import Input from '../components/Objects/Input/Input';
 import homeLogo from "../images/homeLogo.png";
 
-export default function LoginView() {
+function LoginView() {
+  /************************************
+   * State
+   ************************************/
+
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  const [errorMessage, setErrorMessage] = useState(null);
+
+  const [email, setEmail] = useState('');
+  const [emailErrorMessage, setEmailErrorMessage] = useState(null);
+  const [password, setPassword] = useState('');
+  const [passwordErrorMessage, setPasswordErrorMessage] = useState(null);
+
+  /************************************
+   * Helper Functions
+   ************************************/
+
+ /* Handler for log in button clicked, if valid inputs logs in user
+  */
+  async function loginClicked() {
+    if (validateInput()) {
+      await loginUser();
+    }
+  }
+
+ /* Makes REST request to log in user, based on response status will 
+  * redirect to the landing view or display and error message
+  */
+  async function loginUser() {
+    //TODO: Set up a reverse proxy so the backend/frontend can be reached on same port
+    // const url = "http://localhost:8080/api/login"; //will need to use this ran locally
+    const url = "/api/login";
```

Comment:
It might be best to have the urls in a config.json file.

Reply:
Good call